WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Package Details/Package Selection/ShippingLabelPackageList.swift

```diff
@@ -0,0 +1,73 @@
+import SwiftUI
+import Yosemite
+
+struct ShippingLabelPackageList: View {
+    @EnvironmentObject private var viewModel: ShippingLabelPackageDetailsViewModel
+    @Environment(\.presentationMode) var presentation
+
+    var body: some View {
+        ScrollView {
+            VStack(spacing: 0) {
+
+                /// Custom Packages
+                ///
+                if viewModel.customPackages.count > 0 {
```

Comment:
What do you think about having an `showHeader` property. I think it makes the code read more declarative and also allows you to test the condition on the view model.

Reply:
cd745eb424825dac53f3f1b1a0dee6e438173545 👍